tachyon/math/finite_fields/extension_field_base.h

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2022 The Plonky3 Authors
+// Use of this source code is governed by a MIT/Apache-2.0 style license that
+// can be found in the LICENSE-MIT.plonky3 and the LICENCE-APACHE.plonky3
+// file.
+
+#ifndef TACHYON_MATH_FINITE_FIELDS_EXTENSION_FIELD_BASE_H_
+#define TACHYON_MATH_FINITE_FIELDS_EXTENSION_FIELD_BASE_H_
+
+#include <vector>
+
+#include "tachyon/math/finite_fields/extended_packed_field_traits_forward.h"
+#include "tachyon/math/finite_fields/packed_field_traits_forward.h"
+
+namespace tachyon::math {
+
+template <typename Derived>
+class ExtensionFieldBase {
```

Comment:
This class only provides a single static function. Does it really need to be implemented as a class? What if GetExtendedPackedPowers is appropriately implemented elsewhere?

Reply:
Indeed, I originally wanted to implement this as direct function in for extension fields (so we can do ext_f.GetExtendedPackedPowers(size) instead of GetExtendedPackedPowers(ext_f, size)), but since the code is all the same, I added as a single parent class. I agree it's not the ideal place for this function. Do you have any suggestions on where it can be placed instead?